Dockerfile

```diff
@@ -41,7 +41,7 @@ ENV BUILD_ENV ${BUILD_ENVIRONMENT}
 WORKDIR ${APP_HOME}
 
 RUN addgroup --system django \
-    && adduser --system --ingroup django django
+    && adduser --system --ingroup django --home /home/django django
```

Comment:
It works without this, but I've put this in as well to try to preserve editor/public parity.

I don't understand why this doesn't fail the same way any more.